exporter/datadogexporter/examples/collector-metrics.yaml

```
@@ -0,0 +1,27 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'otelcol'
+        scrape_interval: 10s
+        static_configs:
+        - targets: ['0.0.0.0:8888']
+
+processors:
+  batch:
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+
+exporters:
+  datadog:
+    api:
+      site: <DD_SITE>
```

Use an environment variable or a comment:
```suggestion
      site: ${env:DD_SITE}
```

---

Updated everywhere 👍